packages/repository-modernizer/src/restructure-pom.js

```diff
@@ -23,6 +23,7 @@ const {
 const path = require("path");
 const fs = require("fs");
 const fetch = require("node-fetch");
+const { POM_XML } = require("./util/constants");
```

Comment:
`constants` is already imported, so just use `constants.POM_XML`. No need for importing this specifically.